plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/PreferenceInitializer.java

```diff
@@ -11,7 +11,7 @@
   @Override
   public void initializeDefaultPreferences() {
     IPreferenceStore store = Activator.getDefault().getPreferenceStore();
```

Comment:
We'll still have duplicated information whether it's the scope + node in `getPreferenceScope()` or here in the URL passed to `PreferenceResolver`.